docs/UserGuide.md

```diff
@@ -64,6 +64,14 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
+### Instantaneous launching
+Users that have Java 11 or above installed in their computers can launch the Duke The Market program 
+by double clicking on the file.
+
+### Saving the data
+Duke The Market data are saved in the hard disk automatically after any command that changes the data. 
```

Comment:
Same here

Reply:
Feel free to refer to the conversation on #16.